libs/pages/dashboard/src/lib/ui/rename-goup/rename-group.component.html

```diff
@@ -0,0 +1,45 @@
+<armonik-admin-gui-dashboard-group-actions-button (click)="openModal()">
```

Comment:
I feel like this component should permit to add one of the removed items to the group

Reply:
Yes, nice idea. I opened an issue because I will be created later.